search-api/search-actors/src/main/java/org/sunbird/actors/SearchActor.java

```diff
@@ -104,6 +104,18 @@ private SearchDTO getSearchDTO(Request request) throws Exception {
             if (filters.containsKey("relatedBoards"))
                 filters.remove("relatedBoards");
 
+            if (filters.containsKey(SearchConstants.contentType) && null != filters.get(SearchConstants.contentType)) {
+                List<String> contentTypeList = getList(filters.get(SearchConstants.contentType));
+                if ((CollectionUtils.isNotEmpty(contentTypeList) && contentTypeList.size() == 1) && contentTypeList.contains("Course")) {
```

Comment:
The value check should not be case sensitive.

1. Course
2. course
3. COURSE
All are valid for search API.

Reply:
changes done. 
https://github.com/project-sunbird/knowledge-platform/pull/276/files